web/src/components/workday-browser/WeekControl.tsx

```diff
@@ -101,6 +103,13 @@ const WeekControl = () => {
       <IconButton sx={iconButtonSx} onClick={goToNextWeek} aria-label={t("controls.aria.nextWeek")}>
         <ArrowForwardIcon />
       </IconButton>
+      <IconButton
```

Comment:
I think we should wrap this in a `<Tooltip>` since it's just an icon button and in a very prominent place: https://mui.com/material-ui/react-tooltip/#arrow-tooltips

(Actually, this treatment should be applied to the old icon buttons, too 😁 )

Reply:
On another thought, can you add the tooltip to the arrow buttons, too, while we're at it. You can use the existing aria-label translation strings.